packages/server-dev/src/manager/watchers/configWatcher.mjs

```diff
@@ -13,11 +13,24 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 */
+/* eslint-disable no-console */
 
-import setupWatcher from './setupWatcher.mjs';
+import getLowdefyVersion from '../utils/getLowdefyVersion.mjs';
+import setupWatcher from '../utils/setupWatcher.mjs';
 
 async function configWatcher(context) {
-  const callback = async () => {
+  const callback = async (filePaths) => {
+    const lowdefyYamlModified = filePaths
+      .flat()
+      .some((filePath) => filePath.includes('lowdefy.yaml') || filePath.includes('lowdefy.yml'));
```

Comment:
lowdefy.json

Reply:
We don't support lowdefy.json in the CLI. I guess we should?

Reply:
Would be nice.